### DISCUSSION TITLE
Add Nuxt js support

### DISCUSSION THREAD

**Discussion by ghost** [Thinkmill/keystatic#809] [Ideas]
Hello,

Thank you for creating such a great project. I have enjoyed it a lot. I would like to suggest adding support for Nuxt.js to Keystatic. I am interested in helping with this and I would appreciate more information on how to get started. Please let me know your thoughts on this.

Thank you.

**Comment by simonswiss** [comment]
Hey!

Because Keystatic is built with React, I believe the only way to make Keystatic work with Nuxt.js is to have Keystatic in a subdirectory in your Nuxt.js project and run it as a separate Next.js app on a different port.

You can make it store the content at the root level of your project (where the Nuxt.js app lives) and then read the files from there.

The other thing that will be a bit complicated is to render content from the `document` field. Keystatic provides the `DocumentRenderer` written in React — and you could take inspiration to write a similar renderer in Vue.

The rest should work pretty well — if you're happy with that dual-nested-app setup.

Hope it helps!

**Reply by ghost** [reply]
Got it, thank you @simonswiss 👍
